modules/ROOT/pages/index.adoc

```diff
@@ -9,90 +9,14 @@
 
 :console: 0
 
-This is the Neo4j Operations Manual, which includes all the operational details and instructions for installing and deploying Neo4j in a self-hosted environment or in the cloud.
+This is the Neo4j Operations Manual, which includes all the operational details and instructions for installing and deploying Neo4j on-premise and in the cloud.
```

Comment:
```suggestion
This is the Neo4j Operations Manual, which includes all the operational details and instructions for installing and deploying Neo4j on-premises and in the cloud.
```

Reply:
Ah, actually, I think on-premise is the correct word.

Reply:
But 'on-premises' is the term for something located on a site or property, typically in the context of IT infrastructure, such as software or hardware that is hosted locally rather than in the cloud.
This is not correct?
['On-premise' is ](https://en.wikipedia.org/wiki/On-premises_software)

Reply:
I see. I've never heard anyone call it on-premises; always on-prem or on-premise.